user/urls.py

```diff
@@ -1,10 +1,17 @@
 from django.urls import path
+from rest_framework_simplejwt.views import (TokenRefreshView,
+                                            TokenObtainPairView,
+                                            TokenVerifyView)
+
 from user.views import CreateUserView, CreateTokenView, ManageUserView
 
 app_name = "user"
 
 urlpatterns = [
     path("register/", CreateUserView.as_view(), name="create"),
     path("login/", CreateTokenView.as_view(), name="login"),
```

Comment:
you don't need this endpoint as you use JWT auth only

Reply:
Removed